website/src/pages/use-cases/incident-response.tsx

```diff
@@ -0,0 +1,343 @@
+        type: 'Blog post',
+        image: 'https://sourcegraphstatic.com/blog/log4j/log4j-blog-thumbnail.png',
+        href: '/blog/log4j-log4shell-0-day',
+    },
+    {
+        title: `The real weakest link in software supply chain security (it's not open source)`,
+        description:
+            'Using open source code can jump-start development but it can also expose you to security vulnerabilities. In this post, learn how to design an effective vulnerability management process that can make dependencies visible and mitigation less time-consuming.',
+        type: 'Blog post',
+        image: 'https://storage.googleapis.com/sourcegraph-assets/blog/third-party-open-source-vulnerabilities.png',
+        href: '/blog/real-weakest-link-in-software-supply-chain-security',
+    },
+    {
+        title: 'How to remove secrets from your codebase',
+        description:
+            'Back in early 2021, Sourcegraph stored infrastructure and service passwords in private repositories. Security engineer André Eleuterio moved every secret to a secure vault and used code search to ensure the move was successful and complete.',
+        type: 'Blog post',
+        image: 'https://sourcegraphstatic.com/blog/securing-sourcegraph-eliminating-secrets.png',
+        href: '/blog/eliminate-secrets-from-codebase-with-universal-code-search',
+    },
+]
+
+const UseCasePage: FunctionComponent<PageProps> = props => (
+    <Layout
+        location={props.location}
+        meta={{
+            title: 'Incident Response | Sourcegraph',
+            description:
+                'Identify the root cause of an incident, understand its potential impact, fix the issue everywhere in your codebase. Incident response from Sourcegraph.',
+            image: 'https://about.sourcegraph.com/sourcegraph-og.png',
+        }}
+        className="use-cases-page"
+        heroAndHeaderClassName={`${styles.useCaseHeader} navbar-light`}
+        hero={
+            <>
+                <div className="bg" />
+                <div className="container pb-4">
+                    <div className="row">
+                        <div className="col-lg-7 mb-8 mt-7">
+                            <BackButtonBold href="/use-cases" text="USE CASES" />
+                            <h1 className="display-2 font-weight-bold mb-4">
+                                Resolve incidents quickly and confidently
+                            </h1>
+                            <div className="display-4 font-weight-normal mb-5">
+                                Identify the root cause of an incident, understand its potential impact on other
+                                services, and fix the issue everywhere in your codebase so it won't reoccur.
+                            </div>
+                            <div className="d-flex flex-column flex-lg-row pt-1">
+                                <Link
```

Comment:
Can we add button tracking on these CTAs? I notice the other use case pages need them here as well — I can create a separate follow up issue and deal with it — don't worry about that here. Let's just make sure all the CTAs on this page have the correct tracking attributes

Reply:
Great point to include. I added tracking to the following CTA buttons in 3746a6c:
- Hero: Request a demo
- Hero: Try Sourcegraph now
- Body/"Get started with Sourcegraph": Request a demo
- Body/"Respond to incidents faster": Ready to get started?

Are there any others on this page that are appropriate to add tracking to? On other [Use Cases](https://about.sourcegraph.com/use-cases/), it doesn't look like we are tracking links to "Learn more", so I am a bit unsure what is in scope. 

I am happy to support the follow up issue, as well.